### ISSUE TITLE
Kasa KD110 error="unsupported scheme: kasa://........

### ISSUE THREAD

**Issue by alexanderjacuna** [AlexxIT/go2rtc#689]
Attempting to setup my Kasa KD110 doorbell camera and using the following config.yml:

```yaml
mqtt:
  enabled: False

timestamp_style:
  format: "%Y/%m/%d %H:%M:%S"

detectors:
  coral:
    type: edgetpu
    device: pci:0

objects:
  track:
    - person

record:
  enabled: True

snapshots:
  enabled: True

rtmp:
  enabled: False

go2rtc:
  streams:
    doorcam: 
      - "kasa://user%40gmail.com:base64password@10.13.36.214:19443/https/stream/mixed?video=h264&audio=g711&resolution=hd"
      - "ffmpeg:doorcam#video=h264#audio=copy"

cameras:
  doorcam:
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/doorcam?video=copy&audio=copy
          roles:
            - detect
            - record
    detect:
      width: 1920
      height: 1080
```

From what @duidain has found in his environment this works for them. As discussed in issue https://github.com/AlexxIT/go2rtc/issues/666

The error is below. I have tried setting my email to use both "user@gmail.com" and "user%40gmail.com" with the same results. Web interface has the message "No frames have been received, check error logs".

```yaml
2023-10-14 06:38:06.182969452  [2023-10-14 06:38:06] frigate.app                    INFO    : Camera processor started for doorcam: 1216
2023-10-14 06:38:06.182972424  [2023-10-14 06:38:06] frigate.app                    INFO    : Capture process started for doorcam: 1218
2023-10-14 06:38:06.200064733  [2023-10-14 06:38:06] detector.coral                 INFO    : Starting detection process: 1213
2023-10-14 06:38:06.200276474  [2023-10-14 06:38:06] frigate.detectors.plugins.edgetpu_tfl INFO    : Attempting to load TPU as pci:0
2023-10-14 06:38:06.200367366  [2023-10-14 06:38:06] frigate.detectors.plugins.edgetpu_tfl INFO    : TPU found
2023-10-14 06:38:06.206809509  06:38:06.206 ERR github.com/AlexxIT/go2rtc/cmd/streams/producer.go:53 > error="unsupported scheme: kasa://user%40gmail.com:base64password@10.13.36.214:19443/https/stream/mixed?video=h264&audio=g711&resolution=hd" url=kasa://user%40gmail.com:base64password@10.13.36.214:19443/https/stream/mixed?video=h264&audio=g711&resolution=hd
2023-10-14 06:38:06.211272346  06:38:06.211 ERR github.com/AlexxIT/go2rtc/cmd/streams/producer.go:53 > error="unsupported scheme: kasa://user%40gmail.com:base64password@10.13.36.214:19443/https/stream/mixed?video=h264&audio=g711&resolution=hd" url=kasa://user%40gmail.com:base64password@10.13.36.214:19443/https/stream/mixed?video=h264&audio=g711&resolution=hd
2023-10-14 06:38:11.317079239  [2023-10-14 06:38:11] frigate.video                  ERROR   : doorcam: Unable to read frames from ffmpeg process.
2023-10-14 06:38:11.317088678  [2023-10-14 06:38:11] frigate.video                  ERROR   : doorcam: ffmpeg process is not running. exiting capture thread...
2023-10-14 06:38:11.320394019  06:38:11.320 ERR github.com/AlexxIT/go2rtc/cmd/streams/producer.go:53 > error="exec: exit status 1" url=ffmpeg:doorcam#video=h264#audio=copy
2023-10-14 06:38:11.320535166  06:38:11.320 ERR github.com/AlexxIT/go2rtc/cmd/streams/producer.go:53 > error="unsupported scheme: kasa://user%40gmail.com:base64password@10.13.36.214:19443/https/stream/mixed?video=h264&audio=g711&resolution=hd" url=kasa://user%40gmail.com:base64password@10.13.36.214:19443/https/stream/mixed?video=h264&audio=g711&resolution=hd
2023-10-14 06:38:11.325823765  06:38:11.325 ERR github.com/AlexxIT/go2rtc/cmd/streams/producer.go:53 > error="unsupported scheme: kasa://user%40gmail.com:base64password@10.13.36.214:19443/https/stream/mixed?video=h264&audio=g711&resolution=hd" url=kasa://user%40gmail.com:base64password@10.13.36.214:19443/https/stream/mixed?video=h264&audio=g711&resolution=hd
```

**Comment by AlexxIT** [comment]
Unsupported scheme means you using old go2rtc without support this source

**Comment by alexanderjacuna** [comment]
Currently running [1.8.0](https://github.com/AlexxIT/go2rtc/commit/6f2af783925f366f150d9dab1cb2f0430c834871) of go2rtc.

**Comment by alexanderjacuna** [comment]
Still seeing the same issue. Folder has been renamed and given execute permissions.

**Comment by AlexxIT** [comment]
Can you show app version and error in same logs?

**Comment by alexanderjacuna** [comment]
@AlexxIT Wasn't able to find out how to display the go2rtc app version from the docker logs. In around about way I was able to figure out what the issue was from your question though. The binary wasn't mapped to my docker container so the old version of go2rtc was being used instead. Once I mapped the new go2rtc binary it worked perfectly. 

https://docs.frigate.video/configuration/advanced/#custom-go2rtc-version

From the documentation I'd add that if using docker the binary needs to also be mapped. Placing the file in the config directory does nothing if it isn't mapped to docker.

/somehostpath/config/go2rtc:/config/go2rtc
